questdb-rs/src/ingress/mod.rs

```diff
@@ -26,12 +26,17 @@
 //!
```

Comment:
Good changes.

The docs here are getting non-trivial in size.

How about moving them with a

```
#![doc = include_str!("mod.md")]
```

And then move these docs to `questdb/src/ingress/mod.md`.